airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -75,7 +76,7 @@ def get_conn(self):
             )
         return self.conn
 
-    def make_query(self, query, include_deleted=False, query_params=None):
+    def make_query(self, query: str, include_deleted: bool = False, query_params: Optional[Dict] = None) -> Dict:
```

Comment:
```suggestion
    def make_query(self, query: str, include_deleted: bool = False, query_params: Optional[dict] = None) -> dict:
```

We could just keep it as `dict` since that is same as Dict[Any, Any] (and same as Dict)

Reply:
Yup, that will keep things simple!